source/brailleDisplayDrivers/superBraille.py

```diff
@@ -0,0 +1,142 @@
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2017 NV Access Limited, Coscell Kao
+
+import serial
+from collections import OrderedDict
+import braille
+import hwPortUtils
+import hwIo
+import time
+import inputCore
+from logHandler import log
+
+BAUD_RATE = 9600
+TIMEOUT = 0.5
+
+# Tags sent by the SuperBraille
+# Sent to identify the display and receive amount of cells this unit has
+DESCRIBE_TAG = "\xff\xff\x0a"
+# Sent to request displaying of cells
+DISPLAY_TAG = "\xff\xff\x04\x00\x99\x00\x50\x00"
+
+class BrailleDisplayDriver(braille.BrailleDisplayDriver):
+	name = "superBrl"
+	# Translators: Names of braille displays.
+	description = _("SuperBraille")
+	isThreadSafe=True
+	_dev=None

```

Comment:
I don't think this needs to be initialised here; it should be set in the constructor. There's a risk that this might hide a logic error somewhere.